packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte

```diff
@@ -776,20 +838,28 @@
       </div>
     </div>
   {/if}
-  {#if !isPayee && !unsupportedNetwork && !isPaid && !isSigningTransaction}
-    <Button
-      className="pay-button"
-      type="button"
-      text={!correctChain
-        ? "Switch Network"
-        : !approved &&
-            paymentCurrencies[0]?.type === Types.RequestLogic.CURRENCY.ERC20
-          ? "Approve"
-          : "Pay Now"}
-      padding="px-[12px] py-[6px]"
-      onClick={handlePayment}
-    />
-  {/if}
+  <div class="invoice-view-actions">
+    {#if !isPayee && !unsupportedNetwork && !isPaid && !isRequestPayed && !isSigningTransaction}
+      {#if !hasEnoughBalance}
+        <div class="balance-warning">
+          Insufficient funds: {userBalance}
+          {paymentCurrencies[0]?.symbol || "ETH"}
```

Comment:
Don't default to incorrect data. Default to `Unknown` or `-` if you can't determine the symbol.